packages/wallet/backend/tests/mocks.ts

```diff
@@ -308,12 +308,50 @@ export const mockedListGrant = [
   {
     id: faker.string.uuid(),
     client: faker.lorem.slug(),
-    state: 'APPROVED'
+    state: 'APPROVED',
+    access: [
+      {
+        identifier: faker.internet.url(),
+        limits: {
+          receiver: null,
+          debitAmount: {
+            value: '12425',
+            assetCode: 'USD',
+            assetScale: 2
+          },
+          receiveAmount: {
+            value: '12300',
+            assetCode: 'USD',
+            assetScale: 2
+          },
+          interval: 'R/2016-08-23T08:00:00Z/P1M'
```

Comment:
why the mocks are changed but there are no tests to check the interval transformation?

Reply:
It covers the list pagination test